scene/2d/tile_map.cpp

```diff
@@ -894,6 +894,19 @@ int TileMap::get_cellv(const Vector2 &p_pos) const {
 	return get_cell(p_pos.x, p_pos.y);
 }
 
+void TileMap::set_cell_region(const Vector2i &p_top_l, const Vector2i &p_dim, const Vector<int> &p_ids)
```

Comment:
Could you use a Rect2i instead of two Vector2i? It would be more consistent with most of the API when setting regions.